static/js/activity.js

```diff
@@ -279,30 +330,6 @@ exports.update_users = function (user_list) {
     }
     users = filter_and_sort(users);
 
-    function get_num_unread(user_id) {
-        if (unread.suppress_unread_counts) {
-            return 0;
-        }
-        return unread.num_unread_for_person(user_id);
-    }
-
-    // Note that we do not include ourselves in the user list any more.
-    // If you want to figure out how to get details for "me", then revert
-    // the commit that added this comment.
-
-    function info_for(user_id) {
-        var presence = exports.presence_info[user_id].status;
-        var person = people.get_person_from_user_id(user_id);
-        return {
-            name: person.full_name,
-            user_id: user_id,
-            num_unread: get_num_unread(user_id),
-            type: presence,
-            type_desc: presence_descriptions[presence],
-            mobile: exports.presence_info[user_id].mobile,
-        };
-    }
-
```

Comment:
It looks like you're extracting these to be top-level functions; can you rebase to this this in its own commit at the start of the branch (with no code changes) so that we can merge this piece now?  That would help shrink the size of the diff :)